packages/circuit-ui/components/SelectorGroup/SelectorGroup.stories.tsx

```diff
@@ -70,4 +70,26 @@ export const Multiple = (args: SelectorGroupProps) => {
   );
 };
 
-Multiple.args = { ...baseArgs, multiple: true };
+MultipleSelection.args = { ...baseArgs, multiple: true };
+
+export const MultipleLines = () => {
```

Comment:
This structure is different than the one that I see used in the repository. I did this approach specially to use this story for testing. Let me know what you think :)

Reply:
I'm not sure I understand what you mean by "this structure", could you elaborate, please?

Reply:
Nevermind this comment, I was talking about the way that code is organized in Storybook, but in the end there's no big change here